mlkem/reduce.c

```diff
@@ -43,8 +43,17 @@ int16_t montgomery_reduce(int32_t a) {
   // Replace C -> C * q in the above and estimate
   // q / 2^17 < 0.0254.
 
-  t = (int16_t)a * QINV;
-  t = (a - (int32_t)t * KYBER_Q) >> 16;
+  uint16_t u;
+  int16_t t;
```

Comment:
Redeclaration of t here fails to compile.

Reply:
oops.

Reply:
Fixed.